test/integration/tests.js

```diff
@@ -65,6 +65,34 @@ test('Gets content types', async (t) => {
   t.ok(response.items, 'items')
 })
 
+test('Gets a content type that has resource links', async (t) => {
+  t.plan(4);
+  const response = await client.getContentType("catalog");
+
+  t.ok(response.sys, 'sys');
+  t.ok(response.name, 'name');
+  t.ok(response.fields, 'fields');
+  t.deepEqual(response.fields[
```

Comment:
```suggestion
  t.deepEqual(response.fields, [
```

is it missing a comma here? 🤔